src/components/RequestGroupComposition/Configuration.vue

```diff
@@ -138,59 +146,89 @@
               field="dither-orientation"
-              :label="getFromObject(formConfig, ['configuration', 'ditherOrientation', 'label'], 'Orientation')"
-              :desc="getFromObject(formConfig, ['configuration', 'ditherOrientation', 'desc'], '')"
-              :hide="getFromObject(formConfig, ['configuration', 'ditherOrientation', 'hide'], !ditheringIsAllowed)"
+              :label="getFromObject(formConfig, ['configuration', 'dither_orientation', 'label'], 'Orientation')"
+              :desc="
+                getFromObject(
+                  formConfig,
+                  ['configuration', 'dither_orientation', 'desc'],
+                  'Angular rotation of the pattern in degrees measured clockwise East (RA) of North (Declination)'
+                )
+              "
+              :hide="getFromObject(formConfig, ['configuration', 'dither_orientation', 'hide'], !ditheringIsAllowed)"
               :tooltip-config="tooltipConfig"
               :errors="null"
             />
             <custom-field
               v-if="dither.pattern === 'grid'"
               v-model="dither.parameters.numRows"
               field="dither-num-rows"
-              :label="getFromObject(formConfig, ['configuration', 'ditherNumRows', 'label'], 'Number of Rows')"
-              :desc="getFromObject(formConfig, ['configuration', 'ditherNumRows', 'desc'], '')"
-              :hide="getFromObject(formConfig, ['configuration', 'ditherNumRows', 'hide'], !ditheringIsAllowed)"
+              :label="getFromObject(formConfig, ['configuration', 'dither_num_rows', 'label'], 'Number of Rows')"
+              :desc="
+                getFromObject(
+                  formConfig,
+                  ['configuration', 'dither_num_rows', 'desc'],
+                  'Number of offsets in the grid in the RA / horizontal direction'
+                )
+              "
+              :hide="getFromObject(formConfig, ['configuration', 'dither_num_rows', 'hide'], !ditheringIsAllowed)"
               :tooltip-config="tooltipConfig"
               :errors="null"
             />
             <custom-field
               v-if="dither.pattern === 'grid'"
               v-model="dither.parameters.numColumns"
               field="dither-num-columns"
-              :label="getFromObject(formConfig, ['configuration', 'ditherNumColumns', 'label'], 'Number of Columns')"
-              :desc="getFromObject(formConfig, ['configuration', 'ditherNumColumns', 'desc'], '')"
-              :hide="getFromObject(formConfig, ['configuration', 'ditherNumColumns', 'hide'], !ditheringIsAllowed)"
+              :label="getFromObject(formConfig, ['configuration', 'dither_num_columns', 'label'], 'Number of Columns')"
+              :desc="
+                getFromObject(
+                  formConfig,
+                  ['configuration', 'dither_num_columns', 'desc'],
+                  'Number of offsets in the grid in the Declination / vertical direction'
```

Comment:
I think you changed these to Declination from DEC... should we also then change RA to be Right Ascension to be consistent?

Reply:
Yeah I was going back and forth on that one... I switched DEC to Declination because DEC isn't actually an acronym so thought it would make more sense to write it out, and from context I figured everyone would know what RA stood for, so it could stay a bit shorter. But also it's true that if one is written out it might make sense to write out the other... I'll write it out and see what it looks like.